### PR TITLE
Update SDK

### PR DESCRIPTION
**Pull request by RetslySupport**
Issue:
The SDK needs to be updated with the latest changes to the API.

- `Sessions` endpoint has been moved to `platforms/`

Tasks:
- [x] Update sessions endpoint
- [ ] Update sdk version

Deployment:
- [ ] Update Hosted SDK on rets.ly

### CODE REVIEW

**Comment by RongxinZhang** [comment]
@CMClay @Jkhwan please review

**Comment by RongxinZhang** [comment]
blocking: https://github.com/Retsly/www/issues/773

File: retsly.js [hunk 1]
```diff
@@ -2011,4 +2011,4 @@ exports.isBuffer = function (obj) {
 };
 
 },{}]},{},[1])(1)
-});
+});
```
Comment (CMClay) [inline retsly.js]: new line

File: index.js [hunk 2]
```diff
@@ -156,7 +156,7 @@ Retsly.prototype.session = function(cb) {
 
   ajax({
     type: 'GET',
-    url: this.getURL('session'),
+    url: this.host + "platform/session",
```
Comment (CMClay) [inline index.js]: Is this the only place where "platform/" is likely to be seen? It's probably a good idea not to hardcode this url here.

**Comment by CMClay** [comment]
👍

**Comment by Jkhwan** [comment]
👍

**Pull request merged**